src/pgen/cluster/agn_triggering.cpp

```diff
@@ -153,25 +168,26 @@ void AGNTriggering::ReduceColdMass(parthenon::Real &cold_mass,
 
   const bool remove_accreted_mass = remove_accreted_mass_;
 
-  Real md_cold_mass = 0;
+  
 
   parthenon::par_reduce(
       parthenon::loop_pattern_mdrange_tag, "AGNTriggering::ReduceColdGas",
       parthenon::DevExecSpace(), 0, cons_pack.GetDim(5) - 1, kb.s, kb.e, jb.s, jb.e, ib.s,
       ib.e,
       KOKKOS_LAMBDA(const int &b, const int &k, const int &j, const int &i,
-                    Real &team_cold_mass) {
+                    Real &team_cold_mass, Real &team_total_mass) {
         auto &cons = cons_pack(b);
         auto &prim = prim_pack(b);
         const auto &coords = cons_pack.GetCoords(b);
 
         const parthenon::Real r2 =
             pow(coords.Xc<1>(i), 2) + pow(coords.Xc<2>(j), 2) + pow(coords.Xc<3>(k), 2);
         if (r2 < accretion_radius2) {
-
+          const Real cell_total_mass = prim(IDN, k, j, i) * coords.CellVolume(k, j, i);
+          team_total_mass += cell_total_mass;
```

Comment:
As for the `team_cold_mass` below, you should only account for the gas in the interior part (not within the ghost zones).

For reference, we remove the accreted gas in all cells (also within the ghost zones) because there's not boundary exchange afterwards so all the cells need to be consistent in all blocks.